### DISCUSSION TITLE
please fooocus add a support Playground v2.5

### DISCUSSION THREAD

**Discussion by rafiislambd** [lllyasviel/Fooocus#3071] [Ideas]
please update Playground v2.5 support fooocus... thanks

**Comment by mashb1t** [comment]
see https://github.com/lllyasviel/Fooocus/issues/2487, currently not the highest prio.

**Reply by mashb1t** [reply]
@rafiislambd nvm, was easier than expected.
=> Implemented in https://github.com/lllyasviel/Fooocus/pull/3073, please test CFG so we can find the best one for the preset (tested with 2 and 3, both with pro/contra).

**Comment by wisdombox44** [comment]
Playground v2.5 and Stable Cascade works on StableSwarm perfectly. Just need to drop CFG scale 3 to for Playground v2.5
StableSwarm still in beta, some features still under development. Basicaly Fast and image quality is good.
https://github.com/Stability-AI/StableSwarmUI
You can import and use all Fooocus styles to StableSwarm under preset menu.
